components/metrics/metrics_log.cc

```diff
@@ -169,8 +170,10 @@ void MetricsLog::RecordCoreSystemProfile(MetricsServiceClient* client,
 #endif
 
   metrics::SystemProfileProto::OS* os = system_profile->mutable_os();
+#if !defined(STARBOARD)
```

Comment:
Perhaps worth a follow-up bug to look at  enabling `base::SysInfo` module to provide reasonable values, e.g. Starboardize it properly ? Starboard already has relevant APIs and attributes to provide OS name and version.

Reply:
Added a TODO with relevant bug. I will say, it's possible it's not necessary as I imagine a lot of this same info gets logged for us automatically in GEL.